pkg/neptune.pkg

```diff
@@ -11,8 +11,8 @@ Overview is used to generate an overview of all projects.
 Release is used to do neptune-semver conforming releases in an automated manner.
 ''',
     url = 'https://github.com/sociomantic-tsunami/neptune',
-    vendor = 'Sociomantic Labs GmbH',
-    maintainer = 'Sociomantic Labs GmbH <tsunami@sociomantic.com>',
+    vendor = 'dunnhumby Germany GmbH',
+    maintainer = 'dunnhumby Germany GmbH <tsunami@sociomantic.com>',
```

Comment:
What about the email URL here?

Reply:
For now e-mails are not migrated and that's not a legal issue, so it can be fixed in the future (when we actually have a solution for it)